packages/cli/src/databases/migrations/common/1720101653148-AddConstraintToExecutionMetadata.ts

```diff
@@ -108,5 +108,15 @@ export class AddConstraintToExecutionMetadata1720101653148 implements Reversible
 		await context.runQuery(
 			`ALTER TABLE ${executionMetadataTableTemp} RENAME TO ${executionMetadataTable};`,
 		);
+
+		if (context.dbType === 'postgresdb') {
+			// Update sequence so that inserts continue with the next highest id.
+			const tableName = escape.tableName('execution_metadata');
+			const sequenceName = escape.tableName('execution_metadata_temp_id_seq1');
+
+			await context.runQuery(
+				`SELECT setval('${sequenceName}', (SELECT MAX(id) FROM ${tableName}));`,
+			);
```

Comment:
Since there is already a migration for this, why update the existing migration?

Reply:
Currently the up migration may cause an issue and the down migration would not fix it. I thought that if somebody would want to downgrade, then this way they had a way to do it and get back to a working state.

It's not 100% necessary. I can remove it.

Reply:
so if someone ran this migration (from before this PR), and then tried to revert, we want to update the sequence because of the new table has incorrect the sequence value?

Reply:
The root problem is that postgres will not update sequences if you insert rows with IDs. Which is what the initial up migration does and the initial down migration does too.

> so if someone ran this migration (from before this PR), and then tried to revert, we want to update the sequence because of the new table has incorrect the sequence value?

Yes. As I said this applies to a small faction of users that don't want to update to 1.51.0 to fixt his problem, but instead want to downgrade to 1.49.

This part of the PR is less important than fixing the wrong sequence value going forward. So I'd rather remove this then loose time on this.

Reply:
I've removed that change.

Reply:
that makes sense. if you prefer then let's keep the down migration changes. and looks like we also need to merge master back in to fix the e2e tests.

Reply:
I brought it back and I rebased on master. Could you re-approve?